### DISCUSSION TITLE
Variation in miner reward

### DISCUSSION THREAD

**Discussion by PrernaPadhee** [dogecoin/dogecoin#2881] [Q&A]
hi
i observe that for certain dogecoin blocks the miner reward is not equal to block reward + fees , for exmple block 173540, miner reward is 250015 != 250000+20
Can some one please help me understand the cause of this difference?

**Comment by patricklodder** [comment]
Miners can always take less for themselves than subsidy + fees. The rule simply imposes a maximum, not a minimum. Therefore the height of the reward up to that maximum is for the miner to decide in the end - the only way to really know why they took less than they could, is to ask them. Perhaps the miner in question is reading this and can clarify.
